conan.install.profile

```diff
@@ -0,0 +1,23 @@
+[settings]
+os=Windows
+os_build=Windows
+arch=x86_64
+arch_build=x86_64
+compiler=Visual Studio
+compiler.version=16
```

Comment:
Isn't the DbAdapterInterface also adjusted to match C++20? Probably this differentiation is no longer necessary.
Also being the interface header only it should not be needed either (would matter for the TestUtilities).

Reply:
Discussed offline. After some tweeks, it's no longer necessary to have two different conan files